src/Platform/Hosted/extractFeatures.ts

```diff
@@ -27,7 +28,8 @@ export type Features = {
 export function extractFeatures(given = new URLSearchParams(window.location.search)): Features {
   const downcased = new URLSearchParams();
   const set = (value: string, key: string) => downcased.set(key.toLowerCase(), value);
-  const get = (key: string, defaultValue?: string) => downcased.get("feature." + key) ?? defaultValue;
+  const get = (key: string, defaultValue?: string) =>
+    downcased.get("feature." + key) ?? downcased.get(key) ?? defaultValue;
```

Comment:
@southpolesteve This is needed to honor feature flags passed down by the Portal. I see that in #659 you added support for passing down portal feature flags down to DE. However those flags are not prefixed with "feature." so we were not honoring them in DE.

Reply:
Alternatively we could also strip away any `feature.` in each string inside the `set` function so we don't have to handle both cases in `get`. Might be a bit cleaner this way.

@southpolesteve @notlaforge Which one do you prefer?

Reply:
No real preference from me. We can always refactor this later.